index.html

```diff
@@ -824,10 +824,11 @@ <h3>Proofs</h3>
 
           <dt><dfn class="lint-ignore">previousProof</dfn></dt>
           <dd>
-An OPTIONAL string value or unordered list of string values. Each value
-identifies another [=data integrity proof=] that MUST verify before the
-current proof is processed. If an unordered list, all referenced proofs in the
-array MUST verify. This property is used in Section [[[#proof-chains]]].
+The previousProof property is OPTIONAL. If present, it MUST be a string 
+value or unordered list of string values. Each value identifies another 
+[=data integrity proof=] that MUST verify before the current proof is 
+processed. If an unordered list, all referenced proofs in the array 
```

Comment:
I believe all proofs must verify regardless of if the list is ordered or unordered.

Reply:
This part is left as it was but I agree. Anyone else want to weight in? I think the `If an unordered list,` could just be taken off.